docs/getting-started/guides/how-to-understand-strategies-descriptions.md

```diff
@@ -0,0 +1,111 @@
+- Borrowing is when you take money from a lender.
+- Borrowing *against* is used to specify the collateral used by the borrower. Example: "I borrowed 5000$ against my car, so if I don't repay the loan the bank can take it".
+- In DeFi this operation is always collateralized since there are no resources to charge debt.
+
+# DeFi
+
+### Yield Aggregator / Yield Optimizer
+- DeFi protocols like Yearn that provides a single interface that aggregates/optimizes users deposits in a single place and reallocates them in an optimal way.
+### Supply / Deposit / Stake
+- Might be used interchangeably sometimes, but it often means that tokens are sent to a contract external to a [strategy](#strategy).
+- Supply often means that the money is being [lent](#lend--borrow-against)
+### Harvest / Collect / Claim
+- When a [strategy](#strategy) collects [profits](#yield--profit) or [rewards](#rewards) in a blockchain transaction.
+### Sell / Buy
+- Market buy or sell tokens on a Decentralized Exchange like [Uniswap](#uniswap) or Sushiswap.
+### Rewards
+- Tokens earned by a protocol-incentivized action.
+- For example: if you deposit liquidity into Uniswap you get trading fees, but you also get UNI *rewards* for doing this at Uniswap and not at SushiSwap.
+### Lock
+- Tokens that are locked can't be used until a specific condition is met.
+### Unlock
+- Unlocking refers to meeting a condition and receiving locked tokens.
+### Liquidity Provider
+- Liquidity Provider is anyone that invest tokens in a protocol.
+### Joint Liquidity Provider
+- Joint [Liquidity Provider](#liquidity-provider) is when your tokens are aggregated with another person's tokens in order to provide liquidity together.
+### Liquidity Pool
+- A [Liquidity Pool↗](https://www.coindesk.com/learn/what-are-liquidity-pools/) is a decentralized market for 2 (or more) tokens.
+### Stable Pool
+- A Stable Pool is a Liquidity Pool made only of stablecoins.
+### Impermanent Loss
+- When you lose money for being a liquidity provider to liquidity pools when compared to just holding the 2 (or more) tokens in your wallet, it happens when the market price ratio for the tokens diverges from the value that they had when you provided liquidity, which is why it can also be called Divergence Loss.
+### Uniswap
+- The [Uniswap↗](https://app.uniswap.org/#/swap?chain=mainnet↗) protocol, a decentralized exchange that aggregates many Liquidity Pools, used to buy and sell tokens.
+- The Uniswap protocol has a v1, v2 and v3, when you read "Uniswap" it refers to v2, the v3 is always explicitly mentioned like "Uniswap v3".
+### DAI / USDT / USDC
+- The three largest stablecoins in the market, pegged to the dollar value, so 1 token represents 1 dollar.
+### Curve
+- Curve is the most well-known and largest stablecoin swap protocol.
+- CRV is the token it emits and is locked to receive curve protocol [profits](#yield--profit).
```

Comment:
CRV is Curve's gov token. The token is `minted`, and it's not locked. You can lock it afterwards to receive:
- voting power
- boost in the rewards that you get when you provide liquidity
- some fee sharing